cli/release.ts

```diff
@@ -49,13 +52,24 @@ export async function handleRelease({
     try {
       const bundleFilename = await build(manifestFile);
       const manifest = await importManifest<PackVersionDefinition>(bundleFilename);
-      packVersion = manifest.version as string;
+      packVersion = manifest.version as string | undefined;
     } catch (err: any) {
       return printAndExit(`Got an error while building your pack to get the current pack version: ${formatError(err)}`);
     }
   }
 
-  const codaClient = createCodaClient(apiKey, formattedEndpoint);
+  if (!packVersion) {
+    const {items: versions} = await codaClient.listPackVersions(packId, {limit: 1});
+    const [latestPackVersionData] = versions;
```

Comment:
I think we need to handle the case where the are no version yet, can just printAndExit an error message.